lib/middleware/karma.js

```diff
@@ -182,19 +182,20 @@ function createKarmaMiddleware (
               }
             }
 
+            const integrityAttribute = file.integrity ? ` integrity="${file.integrity}"` : ''
+            const crossOriginAttribute = includeCrossOriginAttribute ? ' crossorigin="anonymous"' : ''
             if (fileType === 'css') {
-              scriptTags.push(`<link type="text/css" href="${filePath}" rel="stylesheet">`)
+              scriptTags.push(`<link type="text/css" href="${filePath}" rel="stylesheet"${integrityAttribute}${crossOriginAttribute}>`)
```

Comment:
I wonder if adding `crossorigin` attribute here may break tests if somebody is loading a stylesheet from the CDN or something because of failed CORS check 🤔

It probably shouldn't because the it will send a [simple request](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests) where CORS check is not ran, but would be good to know for sure. @falsandtru Do you know? Or can you check it otherwise?

Reply:
> I wonder if adding crossorigin attribute here may break tests if somebody is loading a stylesheet from the CDN or something because of failed CORS check 

I feel such fallback requests should also be checked by CORS. Otherwise that check should be disabled by configuration.

> It probably shouldn't because it will send a [simple request](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests) where CORS check is not ran, but would be good to know for sure.

I'm also not sure but this should be an appropriate change. So I intend to revert this CORS constraint of link tags if any reasonable use case to disable the constraint without using the configuration is found.

Reply:
Okay, sounds like a reasonable plan to me! I guess the risk of breaking somebody is extremely low here.